### ISSUE TITLE
`lxc image delete` silently ignores extra arguments

### ISSUE THREAD

**Issue by candlerb** [lxc/lxd#2695]
# Required information

 * Distribution: ubuntu
 * Distribution version: 16.04
 * The output of "lxc info"

```
apiextensions:
- id_map
apistatus: stable
apiversion: "1.0"
auth: trusted
environment:
  addresses:
  - 192.168.5.82:8443
  architectures:
  - x86_64
  - i686
  certificate: <snip>
  certificatefingerprint: <snip>
  driver: lxc
  driverversion: 2.0.5
  kernel: Linux
  kernelarchitecture: x86_64
  kernelversion: 4.4.0-47-generic
  server: lxd
  serverpid: 32456
  serverversion: 2.0.8
  storage: zfs
  storageversion: "5"
config:
  core.https_address: '[::]'
  core.trust_password: true
  storage.zfs_pool_name: lxd
public: false
```

# Issue description

`lxc image delete foo bar baz` only deletes image foo (*and* does not complain about bar and baz being superfluous items)

Preferred resolution would be to iterate over the provided arguments and delete them all. This would be consistent with `lxc delete foo bar baz` which deletes all three containers

# Steps to reproduce

```
# lxc image list
+-------+--------------+--------+---------------------------------------------+--------+----------+-----------------------------+
| ALIAS | FINGERPRINT  | PUBLIC |                 DESCRIPTION                 |  ARCH  |   SIZE   |         UPLOAD DATE         |
+-------+--------------+--------+---------------------------------------------+--------+----------+-----------------------------+
|       | 47c261ba095a | no     | Centos 7 amd64 (20161206_02:16)             | x86_64 | 62.72MB  | Dec 6, 2016 at 4:22am (UTC) |
+-------+--------------+--------+---------------------------------------------+--------+----------+-----------------------------+
|       | fc6d723a6e66 | no     | ubuntu 16.04 LTS amd64 (release) (20161130) | x86_64 | 142.35MB | Dec 1, 2016 at 1:50pm (UTC) |
+-------+--------------+--------+---------------------------------------------+--------+----------+-----------------------------+

# lxc image delete 47c261ba095a fc6d723a6e66

# lxc image list
+-------+--------------+--------+---------------------------------------------+--------+----------+-----------------------------+
| ALIAS | FINGERPRINT  | PUBLIC |                 DESCRIPTION                 |  ARCH  |   SIZE   |         UPLOAD DATE         |
+-------+--------------+--------+---------------------------------------------+--------+----------+-----------------------------+
|       | fc6d723a6e66 | no     | ubuntu 16.04 LTS amd64 (release) (20161130) | x86_64 | 142.35MB | Dec 1, 2016 at 1:50pm (UTC) |
+-------+--------------+--------+---------------------------------------------+--------+----------+-----------------------------+
```

**Comment by stgraber** [comment]
We've had that for a while in the recent LXD feature releases.

Probably would be fine to include in the stable 2.0 branch too, not sure why we didn't do that initially.

```
stgraber@castiana:~$ lxc image list
+-------+--------------+--------+---------------------------------------+--------+----------+-----------------------------+
| ALIAS | FINGERPRINT  | PUBLIC |              DESCRIPTION              |  ARCH  |   SIZE   |         UPLOAD DATE         |
+-------+--------------+--------+---------------------------------------+--------+----------+-----------------------------+
|       | 5ae784baa2dd | no     | Alpine 3.1 amd64 (20161205_17:50)     | x86_64 | 2.32MB   | Dec 6, 2016 at 4:26pm (UTC) |
+-------+--------------+--------+---------------------------------------+--------+----------+-----------------------------+
|       | 0f0b7b6b4421 | no     | Alpine edge amd64 (20161205_19:21)    | x86_64 | 2.37MB   | Dec 6, 2016 at 4:26pm (UTC) |
+-------+--------------+--------+---------------------------------------+--------+----------+-----------------------------+
|       | 5662931ad1e4 | no     | ubuntu 17.04 amd64 (daily) (20161205) | x86_64 | 147.95MB | Dec 5, 2016 at 9:37am (UTC) |
+-------+--------------+--------+---------------------------------------+--------+----------+-----------------------------+
stgraber@castiana:~$ lxc image delete 5ae784baa2dd 0f0b7b6b4421
stgraber@castiana:~$ lxc image list
+-------+--------------+--------+---------------------------------------+--------+----------+-----------------------------+
| ALIAS | FINGERPRINT  | PUBLIC |              DESCRIPTION              |  ARCH  |   SIZE   |         UPLOAD DATE         |
+-------+--------------+--------+---------------------------------------+--------+----------+-----------------------------+
|       | 5662931ad1e4 | no     | ubuntu 17.04 amd64 (daily) (20161205) | x86_64 | 147.95MB | Dec 5, 2016 at 9:37am (UTC) |
+-------+--------------+--------+---------------------------------------+--------+----------+-----------------------------+
```

**Comment by stgraber** [comment]
I pushed the fix to stable-2.0

**Comment by candlerb** [comment]
Thank you!
